### ISSUE TITLE
Improve options for viewing/editing uploaded ontologies 

### ISSUE THREAD

**Issue by mmisw** [mmisw/mmiorr#330]
_From [jbgrayb...@mindspring.com](https://code.google.com/u/106719590578390745654/) on October 06, 2014 15:25:33_

What capability do you want added or improved? The ability to work with uploaded ontologies in the simple form available for tool-produced ontologies. Where do you want this capability to be accessible? In editing and viewing UIs. What sort of input/command mechanism do you want? For now, either as an option in each applicable UI as needed; or as a setting that can be made in the ontology.
Someday (in my land of fantasy), as a user setting for each ontology. What is the desired output (content, format, location)? The user can use the editor and viewer on a submitted ontology, and have it work as if the ontology is tool-created or submitted, at the user's discretion. Other details of your desired capability? Currently setting the omv:usedOntologyEngineeringTool property in the ontology, then submitting it, appears not to work (it looks like the new attribute just got lost, using the 'new values overwriting' option). This would be a tolerable workaround. What version of the product are you using? ORR Portal 2.1.1 (201408221430 Please provide any additional information below (particular ontology/ies, text contents of vocabulary (voc2rdf), operating system, browser/version (Firefox, Safari, Chrome, IE, etc.), screenshot, etc.)

_Original issue: http://code.google.com/p/mmisw/issues/detail?id=330_

**Comment by mmisw** [comment]
_From [jbgrayb...@mindspring.com](https://code.google.com/u/106719590578390745654/) on October 06, 2014 15:55:59_

The specific line I added was in the top Ontology rdf:About="" section, and was
   \<omv:usedOntologyEngineeringTool>omvmmi:voc2rdf</omv:usedOntologyEngineeringTool>

**Comment by mmisw** [comment]
_From [jbgrayb...@mindspring.com](https://code.google.com/u/106719590578390745654/) on October 06, 2014 15:56:49_

Oh yeah. And I also tried it with 'use all values from new upload', and that didn't work either.

**Comment by mmisw** [comment]
_From [jbgrayb...@mindspring.com](https://code.google.com/u/106719590578390745654/) on October 16, 2014 21:40:02_

Now that `#331` is fixed I encouraged Emilio to try this again. He reports:

YES!! It worked! Thanks for encouraging me to try it. I just took my previous rdf version and added the element
\<omv:usedOntologyEngineeringTool rdf:resource=" http://mmisw.org/ont/mmi/20081020/ontologyMetadata/voc2rdf"/ >

under rdf:RDF/owl:Ontology

Everything just worked off the bat. The problem covered by issue #331 was fixed (as expected). And the one covered by issue #330 also worked w/o hiccups. The ontology now behaves exactly the way I wanted it, like a simple tabular vocabulary. And when editing a new version, the vocabulary content is now editable in the way I was looking for, via the ORR.

\---

With this workaround available, I've signed this off as Verified (by Emilio).

**Status:** Verified
